stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -214,7 +244,7 @@ internal class PaymentSheetViewModel internal constructor(
         val paymentSelection = selection.value
 
         if (paymentSelection is PaymentSelection.GooglePay) {
-            paymentIntent.value?.let { paymentIntent ->
+            (stripeIntent.value as? PaymentIntent)?.let { paymentIntent ->
```

Comment:
stripeIntent will always be PaymentIntent.   This is should be protected by isGooglePayEnabled in the args

Reply:
It still needs to be cast here, and it's nullable.